jmx-scraper/src/integrationTest/java/io/opentelemetry/contrib/jmxscraper/target_systems/TargetSystemIntegrationTest.java

```diff
@@ -196,4 +198,27 @@ public void export(
       sb.http(0);
     }
   }
+
+  protected static String genericJmxJvmArguments(int port) {
```

Comment:
Good change! However it is a bit over-engineered in my opinion. Simple StringBuilder and additional private method would be enough, no need for maps and streams. It is just my opinion :-)

Reply:
Yeah, I think I agree...I would have just hard coded a bunch of string concats...but I don't think it's a huge problem.